src/app/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.scss

```diff
@@ -9,3 +9,7 @@ form {
 .btn:nth-child(2) {
   margin: 0 0 0 20px;
 }
+
+.validation {
+  color: #f00;
+}
```

Comment:
use some var for this color

Reply:
variable for a color has been added with this commit ffc171f9490026bb04798d1380ae2744aa79012c